### ISSUE TITLE
Cannot update firebase-tools past version 3.19.1

### ISSUE THREAD

**Issue by iinewmanii** [firebase/firebase-tools#873]
### Version info
v3.19.1

### Platform Information
Mac OS 10.13.6

### Steps to reproduce
Stuck on version 3.19.1.

### Expected behavior
When I run npm install -g firebase-tools I expect it to update.

### Actual behavior
When I run firebase --version it still says 3.19.1

**Comment by mbleigh** [comment]
What happens if you do:

    npm i -g firebase-tools@latest

Also, when it is installing what version does it say is installing?

Best guess is you may have multiple global npm directories and you're not
installing into the correct one.

**Comment by iinewmanii** [comment]
If I do @latest it appears that it is installing and it gives me this:

/usr/local/Cellar/node/8.7.0/bin/firebase -> /usr/local/Cellar/node/8.7.0/lib/node_modules/firebase-tools/bin/firebase
/usr/local/Cellar/node/8.7.0/lib
└── firebase-tools@4.1.1
version 4.1.1

but when I use the command firebase --version it still says 3.19.1

When I get off of work I will investigate whether I have more than one directory.

Thank you for your suggestions.

**Comment by iinewmanii** [comment]
I am only finding one firebase-tools folder on my computer. It is in the functions folder of my firebase project. I am using firestore, functions and hosting for my project.

**Comment by mbleigh** [comment]
What do you get when you run

    which firebase
    which node
    which npm

**Comment by iinewmanii** [comment]
/usr/local/bin/firebase
/usr/local/bin/node
/usr/local/bin/npm

**Comment by iinewmanii** [comment]
Thank you for your help with getting the tools updated. Now to figure out why hosting cannot find my css and image files anymore.

**Comment by EmNudge** [comment]
Had a similar problem, was on firebase-tools version 3.19.3 and couldn't get past. Tried uninstalling and the `firebase` command still worked, despite npm letting me know it was uninstalled. Even went as far as downloading the folders so that the command didn't work anymore. Installing firebase again didn't help, the command was still unavailable.

Seems all people have this problem are also using a Mac. Found a thread saying node versions downloaded with homebrew are kinda bugged. I uninstalled node and npm entirely. Downloaded it from the downloads page. Installed firebase globally again.

And then it worked.

TL;DR
uninstall and reinstall node from the downloads page. You may have installed node originally using homebrew and have a bugged version of node.
